### ISSUE TITLE
Init fresh database fails

### ISSUE THREAD

**Issue by spwoodcock** [hotosm/fmtm#836]
## Issue

When clearing and starting from a fresh database, I get error generating the tables:

```
2023-09-20 17:51:30.331 UTC [71] ERROR:  duplicate key value violates unique constraint "pg_type_typname_nsp_index"
2023-09-20 17:51:30.331 UTC [71] DETAIL:  Key (typname, typnamespace)=(userrole, 2200) already exists.
2023-09-20 17:51:30.331 UTC [71] STATEMENT:  CREATE TYPE userrole AS ENUM ('MAPPER', 'ADMIN', 'VALIDATOR', 'FIELD_ADMIN', 'ORGANIZATION_ADMIN', 'READ_ONLY')
```

Needs further testing.

**Comment by spwoodcock** [comment]
Likely solved by #441 (or the following updated PR).

**Comment by susmina94** [comment]
@spwoodcock can you confirm and close this ?
